src/pages/Layout.jsx

```diff
@@ -27,21 +25,9 @@ const Layout = () => {
       <Box component="main" sx={{ flexGrow: 1, p: 3 }}>
         <Toolbar variant="tall" />
         <Outlet />
-        <Box
-          sx={{
-            display: "flex",
-            justifyContent: "center",
```

Comment:
Might be a necessary code in line 27 if you consider moving  Footer to between line 29 and 30

Reply:
Ok, thanks